src/model.c

```diff
@@ -136,9 +138,32 @@ static void RAI_Model_AofRewrite(RedisModuleIO *aof, RedisModuleString *key, voi
     array_append(outputs_, RedisModule_CreateString(ctx, model->outputs[i], strlen(model->outputs[i])));
   }
 
+  char backend[256];
```

Comment:
char backend[256] = "";
to prevent catastrophe if model->backend isn't matched.

Reply:
Done